stan/math/prim/meta/require_generics.hpp

```diff
@@ -799,6 +834,30 @@ template <template <class...> class TypeCheck, class... Check>
 using require_all_not_eigen_vt
```

Comment:
This really needs doc because I have no idea what the `v` is supposed to signal in `vt`.